ballisticbutterflies/The-Paw
# F 12 b

### TESTING INSTRUCTIONS!


To test create a biz BE we must login first.

http://localhost:5000/api/auth/login

{
  "credential": "demo@aa.io",
  "password": "password"
}

 now go to http://localhost:5173/api/businesses/
 
 Do a post request with the required fields:
{
    "address": "123 Test Ln",
    "city": "Test",
    "state": "AL",
    "zip_code": "12345",
    "name": "Test",
    "description": "Test description as it is required."
}

This should successfuly add a biz to your local DB
and you should see it added to the list at http://localhost:5173/search/ in your browser

... remember to remove this instance from your dev.db before pushing it lol 

just go to your source control tab and next to dev.db click discard changes!
